### DISCUSSION TITLE
<title>

### DISCUSSION THREAD

**Discussion by goddess444** [community/community#79282] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I’m hacked I didn’t add data in here

**Comment by hola-aryan** [comment]
What is this 😠

**Comment by goddess444** [comment]
I already stated I have been hacked and need anything  from my account wipedSent from my hacked iPhone On 17 Dec 2566 BE, at 5:29 am, Aryan Mishra ***@***.***> wrote:﻿
What is this 😠

**Comment by queenofcorgis** [comment, marked as answer]
Hi @goddess444 , Thanks for reaching out! 

We're sorry to hear you're experiencing this issue.

GitHub is a software hosting service, we host millions of projects, so probably every phone and computer on the planet has software on it that is also hosted here.

We're not able to provide support or answers for those individual projects. GitHub Support can only provide support for GitHub itself so we'll be closing and locking the discussion.

I'm sorry we can't help here but we'd recommend contacting a local IT specialist for assistance if you think your devices are affected by viruses or malware. They should be able to help you with these applications!
